Makefile

```diff
@@ -12,9 +20,10 @@ ssh:
 	-e "AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}" \
 	-e "AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}" \
 	-e "AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}" \
+	-e MOBILIZE_AMERICA_API_KEY \
```

Comment:
Hm, we need something more flexible here. This could get out of control if we add all the environment variables from each lambda function here. The variables currently here were meant to be for deployment/serverless management.